demos/python_demos/bert_question_answering_embedding_demo/README.md

```diff
@@ -0,0 +1,124 @@
+
+## How It Works
+
+Upon the start-up the demo application reads command line parameters and loads network(s) to the InferenceEngine.
+It also fetches data from the user-provided urls to populate the list of "contexts" with the text.
+Prior to the actual inference to answer user's questions, the embedding vectors are pre-calculated (via inference) for each context from the list.
+This is done using the first ("emdbeddings-only") BERT model.
+
+After that, when user type the question and the "embeddings" network is used to calculate an embedding vector for the specified question.
+Using the L2 distance between the embedding vector of the question and the embedding vectors for the contexts the best (closest) contexts are selected
+as candidates to further seek for the final answer to the question. At this point, the contexts are displayed to the user.
+
+Notice that question is usually much shorter than the contexts, so calculating the embedding for that is really fast. Also calculating the L2 distance between a context and question is almost free, compared to the actual inference. Together, during question answering, this substantially saves on the actual inference, which is needed ONLY for the question (while contexts are pre-claculated), compared to the conventional approach that has to concatenate each context with the question and do an inference on this large input (per context).
+
+If second (conventional SQuAD-tuned) Bert model is provided as well, it is used to further search for the exact answer in the best contexts found in the first step, and the result then also displyed to the user.
+
+## Running
+
+Running the application with the `-h` option yields the following usage message:
+```
+python3 question_answering_embedding_demo.py -h
+```
+The command yields the following usage message:
+```
+usage: question_answering_embedding_demo.py [-h] -i INPUT [--par_num PAR_NUM]
+                                            -v VOCAB --model_emb MODEL_EMB
+                                            [--model_qa MODEL_QA]
+                                            [--input_names INPUT_NAMES]
+                                            [--output_names OUTPUT_NAMES]
+                                            [-a MAX_ANSWER_TOKEN_NUM]
+                                            [-d DEVICE] [-c]
+
+Options:
+  -h, --help            Show this help message and exit.
+  -i INPUT, --input INPUT
+                        Required. Urls to a wiki pages with context
+  --par_num PAR_NUM     Optional. Number of contexts filtered in by embedding
+                        vectors
+  -v VOCAB, --vocab VOCAB
+                        Required. Path to vocabulary file with tokens
+  --model_emb MODEL_EMB
```

Comment:
We prefer `-m` prefix for model keys. I suggest you change it to `-m_emd` etc.

Reply:
do you mean --model_emb is replaced by  --m_emb or new additional key --m_emb should be added??

Reply:
I've made --m_emb and --model_emb together to be consistemt with bert_question_answering_demo